docs/release_notes/v0.11.0.md

```diff
@@ -0,0 +1,489 @@
+
+**Fixed** broken links to binding docs (Thanks @awolf)
+
+**Added** docs for changes to local secret store 
+
+**Updated** actors API reference doc
+
+**Updated** pub/sub API reference doc
+
+**Updated** state API reference doc
+
+**Fixed** broken links (Thanks @sau-lanvy)
+
+**Added** cross namespace information for service invocation 
+
+**Added** declarative pub/sub documentation 
+
+**Updated** documentation for Prometheus and Grafana setup
+
+**Added** metrics spec information 
+
+**Added** disclaimer about plain text secrets 
+
+**Added** Redis failover guidelines (Thanks @abserari)
+
+**Added** secrets scoping information 
+
+**Fixed** missing line continuation in commands 
+
+**Added** configuration options doc for decodeBase64 (Thanks @mtaghavi2005)
+
+**Updated** Helm chart links 
+
+**Updated** NATS streaming setup and deleted NATS setup (Thanks @abserari)
+
+**Added** case-sensitivity of input parameters (state keys, pubsub topics)
+
+**Added** clarification on use of CloudEvents during publish 
+
+**Updated** Azure Blob Storage binding documentation 
+
+**Refctored** service invocation and state management cocncepts doc
+
+**Updated** KEDA HowTo doc
+
+## Upgrading to Dapr 0.11.0
+
+If you're upgrading from an older version of Dapr to 0.11.0, follow the steps here to ensure a smooth upgrade. You know, the one where you don't get red errors on the terminal.. we all hate that, right?
+
+### Local Machine / Self-hosted
+
+Uninstall Dapr using the CLI you currently have. Note that this will remove the default $HOME/.dapr directory, binaries and all containers dapr_redis, dapr_placement and dapr_zipkin. Linux users need to run sudo if they have installed binary in default path /usr/local/bin/ or have docker command needing sudo.:
+
+```bash
+dapr uninstall --all
+```
+
+Next, get the latest CLI following [these](https://github.com/dapr/cli#installing-dapr-cli) instructions, or alternatively download the latest and greatest release from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
+
+Once you have downloaded the CLI, run:
+
+```bash
+dapr init
+```
+
+Wait for the update to finish, and you're good to go!
+Make sure you have the right runtime version (0.11.0) with:
+
+```bash
+dapr --version
+```
+
+### Kubernetes
+
+#### Upgrading from 0.10.0
+
+If you previously installed Dapr using Helm, starting with this releases you can now upgrade Dapr to a new version.
+If you installed Dapr using the CLI, go [here](#starting-fresh).
+
+##### 1. Get the latest CLI
+
+Get the latest version of the Dapr CLI as outlined above, and put it in your path.
+You can also use the helper scripts outlined [here](https://github.com/dapr/cli#installing-dapr-cli) to get the latest version.
+
+##### 2. Upgrade
+
+First, update your Helm repos:
+
+```
+helm repo update
+```
+
+Run the following command to upgrade Dapr:
+
+```
+helm upgrade dapr dapr/dapr --version 0.11.0 --namespace dapr-system --reset-values
```

Comment:
The upgrade outlined here will incur downtime. Is there a reason we are not showing the zero downtime upgrade path here?

Reply:
Will there be any issues with the ClusterRole change?

Reply:
There shouldn't be as we are not actually changing any adding or removing permissions.

Reply:
```
Error: UPGRADE FAILED: cannot patch "dapr-operator" with kind ClusterRoleBinding: ClusterRoleBinding.rbac.authorization.k8s.io "dapr-operator" is invalid: roleRef: Invalid value: rbac.RoleRef{APIGroup:"rbac.authorization.k8s.io", Kind:"ClusterRole", Name:"dapr-operator-admin"}: cannot change roleRef
```
@yaron2 
Getting this error while following the steps defined in https://github.com/dapr/docs/tree/master/howto/deploy-k8s-prod#upgrading-dapr-with-helm

Reply:
See here: https://github.com/dapr/dapr/pull/2119#issuecomment-700328673